chore(moderation: add note about spam posts (#611)
nodejs/admin: Moderation-Policy.md

  * report accounts to GitHub
* Accounts that are reasonably believed to be bots (other than bots authorized
  by both TSC and CommComm) are subject to immediate Blocking.

> * Issues, pull requests, discussions and comments that are considered spam by the Moderation Team
>   (such as job posting, service advertising, etc.) will be deleted.

Reviewer: For consistency with the other items in the list:

```suggestion
* Issues, pull requests, discussions, and comments that are spam (job posting,
  service advertising, etc.) are subject to Immediate moderation.
```

Reviewer 2: lower case "immediate"?

Reviewer: > lower case "immediate"?

Yes. Fixed. Thanks!
